main.go

```diff
@@ -182,18 +182,16 @@ func generateTargetArtifacts(w http.ResponseWriter, r *http.Request) {
 	name := mux.Vars(r)["name"]
 	plan := r.FormValue("plan")
 
-	helmTargetArtifactTypeValue := "Helm"
-	yamlsTargetArtifactTypeValue := "Yamls"
-	knativeTargetArtifactTypeValue := "Knative"
-
-	re := regexp.MustCompile(`artifacttype:\\s.*`)
-	artifactTypematch := re.FindString(plan)
-	artifactType := yamlsTargetArtifactTypeValue
-	if strings.HasSuffix(artifactTypematch, knativeTargetArtifactTypeValue) {
-		artifactType = knativeTargetArtifactTypeValue
-	} else if strings.HasSuffix(artifactTypematch, helmTargetArtifactTypeValue) {
-		artifactType = helmTargetArtifactTypeValue
+	re := regexp.MustCompile(`artifactType:\s*(.+)`)
```

Comment:
```suggestion
	re := regexp.MustCompile(`artifactType:\s*([^\s]+)`)
```
Won't this end up choosing a contiguous set of non space characters like what we want? The issue with the previous regex I had suggested was the user of `*` which was incorrect.

Reply:
If there would be no whitespace in the words between, [^\s]+ can give the correct answer. But if the string is like: 
"artifactType:  he l m", it will only take "he", not "helm". If "he" is what you expected, I can change it to '[^\s]+'.
Please let me know, thanks.